src/actions.py

```diff
@@ -421,3 +423,26 @@ def add_note(args: list[str], notes_book: "NoteBook") -> None:
     notes_book.add(note_title, note)
     notes_book.attach_to_contact(note_title, name)
     print_to_console("Note added.")
+
+
+@input_error
+def cleanup(args: list[str]) -> str:
+    """Cleans up dumps files from system.
+
+    return: str: Result message.
+    """
+    config = args[0]
+
+    files_to_delete = {
+        "all": [ADDRESS_BOOK_FILE, NOTES_FILE],
+        "address-book": [ADDRESS_BOOK_FILE],
+        "notes": [NOTES_FILE],
+    }
+    if files_to_delete.get(config) is None:
+        raise NotFoundWarning("Invalid config value. Choose from 'all', 'address-book', or 'notes'.")
```

Comment:
please change slightly message
https://github.com/vp-mx/pocketpal/blob/af1dd1e1af80154169eacbd073cd1cfed758db8c/src/error_handlers.py#L44

e.g. Choose from 'all', 'address-book', or 'notes'.  Files for {config}

as prefix not found will be appended automatically

Reply:
Fixed